pom.xml

```diff
@@ -80,6 +80,7 @@
         <module>emf-trace</module>
         <module>emf-types</module>
         <module>emf-widespread</module>
+        <module>smm</module>
```

Comment:
Rule (that should be written someday): use "emf" as a prefix for models that use the standard EMF code generation

Reply:
Good point, I'll change that.